include/ebpf_program_types.h

```diff
@@ -28,13 +28,19 @@ typedef struct _ebpf_program_type_descriptor
     char is_privileged;
 } ebpf_program_type_descriptor_t;
 
+typedef enum _ebpf_helper_function_flags
+{
+    EBPF_HEPER_FUNCTION_FLAGS_UNWIND_ON_SUCCESS = 1,
```

Comment:
you have a typo "HEPER" and fyi, I'm using the value 1 for the "reallocates_packet" flag, which will be called
#define EBPF_HELPER_FLAG_REALLOCATE_PACKET 0x01
in my pending prevail PR. So please use consistent name prefix and avoid the value 1.